### ISSUE TITLE
Fluctuating training loss

### ISSUE THREAD

**Issue by 299792459b** [ming024/FastSpeech2#206]
![image](https://github.com/ming024/FastSpeech2/assets/126702234/b61ceb43-259e-40a9-b993-b54d415da990)

Red is training loss
Orange is valid loss

Other datasets show same behavior

Training loss fluctuates up and down drastically.  Anyone know why is this happening? Thank you

**Comment by melodyze-ai** [comment]
Try lowering learning rate.

**Comment by asarsembayev** [comment]
> Try lowering learning rate.

which parameter in config is actually LR?
